sql/as_of_test.go

```diff
@@ -59,13 +61,25 @@ func TestAsOfTime(t *testing.T) {
 		t.Fatal("unexpected error:", err)
 	}
 
-	if _, err := db.Exec("CREATE DATABASE d"); err != nil {
+	var logicalTS int64
+	if err := db.QueryRow("CREATE DATABASE d; SELECT cluster_logical_timestamp()::int").Scan(&logicalTS); err != nil {
 		t.Fatal("unexpected error:", err)
 	}
-	if err := db.QueryRow("SELECT now()").Scan(&tm); err != nil {
-		t.Fatal(err)
-	}
-	tsDBExists := tm.Format(time.RFC3339Nano)
+	// Make sure we see the DB create by using a TS after its txn time.
```

Comment:
Dunno yet. This is a stop gap to prevent the test flakiness. I've opened #9899 to track that bug.